todo.md

```diff
@@ -0,0 +1,26 @@
+# TODO
```

Comment:
Can we not have this in the main repo, please? If you want to add a todo list for the SyncBot spec., that's fine but not in the code.